lib/perl/Genome/Model/Build/ClinSeq/InputBuilds.pm

```diff
@@ -30,4 +30,9 @@ sub resolve_rnaseq_builds {
   $rnaseq_builds->{$normal_build->id}{type} = 'rnaseq' if $normal_build;
 }
 
+sub wgs_exome_build {
```

Comment:
Are there any suggestions for improving the consistency between these subroutines?  I'm not entirely sure what the rest of ClinSeq expects these to return.  There is a future task to refactor how ClinSeq resolves all of these input properties:
https://jira.gsc.wustl.edu/browse/CI-109